service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasApplicantContactNameValidator.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import org.apache.commons.lang.StringUtils;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasApplicantContactName;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasApplicantContactNameValidator implements ConstraintValidator<HasApplicantContactName, ApplicantParty> {
+    @Override
+    public void initialize(HasApplicantContactName constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(ApplicantParty applicantParty, ConstraintValidatorContext constraintValidatorContext) {
+        return applicantParty.getTelephoneNumber() != null
+            && applicantParty.getTelephoneNumber().getContactDirection() != null
+            && StringUtils.isNotBlank(applicantParty.getTelephoneNumber().getContactDirection());
```

Comment:
Aren't you are already checking the != null in the isNotBlank part?

Reply:
I am checking `applicantParty.getTelephoneNumber.GetContactDirection()` is not null in the isNotBlank part. Method will give null exception without.

Reply:
Good spot